src/ports/catalog/queries.ts

```diff
@@ -465,65 +600,76 @@ export const getCollectionsItemsCatalogQuery = (filters: CatalogQueryFilters) =>
               items.reviewed_at,
               items.sold_at,
               items.network,
-              items.first_listed_at,
+              GREATEST(items.first_listed_at, ROUND(EXTRACT(EPOCH FROM offchain_orders.max_created_at))) as first_listed_at,
```

Comment:
should this be greatest? the first_listed_at should be the least of this dates right?